From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paul Gofman <pgofman@codeweavers.com>
Date: Wed, 30 Jul 2025 12:07:10 -0600
Subject: [PATCH] msi: HACK: Report older Windows version for Warhammer 40,000:
 Dawn of War III redists.

CW-Bug-Id: #25520
Valve-commit: 3b8e5cb712d6a33cd590529d0b9cc4022d6ba576
Picked: clean
Class: valve
Evidence: docs/evidence/2026-09-28-wine-proton-rebase.md
Offered-upstream: no
---
 dlls/msi/package.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/msi/package.c b/dlls/msi/package.c
index 5db4c10..04009d0 100644
--- a/dlls/msi/package.c
+++ b/dlls/msi/package.c
@@ -635,6 +635,7 @@ static void fixup_winver(DWORD *verval)
                         !strcmp(s, "976730")
                         || !strcmp(s, "231430")
                         || !strcmp(s, "1017900")
+                        || !strcmp(s, "285190")
                     );
         if (cached)
             ERR("HACK: setting winver 502.\n");
